boards/arm/usb_kw24d512/usb_kw24d512_defconfig

```diff
@@ -0,0 +1,12 @@
+CONFIG_ARM=y
+CONFIG_SOC_MKW24D5=y
+CONFIG_SOC_SERIES_KINETIS_KWX=y
+CONFIG_BOARD_USB_KW24D512=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_CONSOLE_ON_DEV_NAME="UART_0"
```

Comment:
I've changed it and just added J-Link support. Once the support for kw24d5 is in pyOCD or OpenOCD mainline, I will expand the docu.